app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md

```diff
@@ -0,0 +1,182 @@
+---
+title: When Franics felt frustrated
+description: We developed a story about subject knowledge enhancement courses. The purpose of the story is to show how we can empathise with users by capturing their pain points in an engaging way.
+date: 2023-07-12
+tags:
+  - subject knowledge enhancements
+ogImage:
+  src: francis-cover-image.png
+  alt: "Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews."
+---
+
+![Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews.](francis-cover-image.png)
+
+Previously, we designed guidance to tell candidates they have to do a [subject knowledge enhancement (SKE) course](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.
+
+We tested these designs and learnt a lot about SKE courses. Some of the findings we uncovered were about the wider SKE policy and do not fit with Apply for teacher training (Apply).
+
+We wanted to capture these findings in a way that would help people empathise with candidates and providers. So, we wrote a story....
+
+We’ve added the story as a design history to share with other teams. It's purpose is to think about different ways to present research and show empathy beyond individual services.
```

Comment:
I feel like this paragraph could be dropped?  The previous paragraph has a nice lead-in to the story already.

```suggestion
```